doc/source/examining/loading_data.rst

```diff
@@ -558,7 +558,7 @@ using a ``parameters`` dict, accepting the following keys:
 AMReX / BoxLib Data
 -------------------
 
-AMReX and BoxLib share a frontend (currently named ``boxlib``), since
+AMReX and BoxLib share a frontend, since
```

Comment:
Isn't this sentence missing a verb ?

Reply:
oh ya! good catch, might as well fix that too :)

Reply:
(but will wait to see how the docs build goes)

Reply:
verb added to the next line!